content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -117,9 +117,9 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
 
 {{< feature-state state="beta" for-kubernetes-version="1.12" >}}
 
-A DaemonSet ensures that all eligible nodes run a copy of a Pod. Normally, the
+A DaemonSet resource type ensures that all eligible nodes run a copy of a Pod. Normally, the
```

Comment:
Why this change? Because DaemonSet is written with titlecase it looks like a type of Kubernetes object, and it is.

Reply:
Since the previous section only had references to DaemonSet controller, I wanted to be explicit and re-center the conversation to make it clear we were now once again talking about the DaemonSet resource type rather than just the controller, let me know if you prefer it without

Reply:
Previously Kubernetes talked about “ReplicationController” etc, as if the control loop were more important than the resource. Things are moving to talk about ReplicaSet, Deployment, DaemonSet etc rather than “a ReplicaSet controller”. The main exception to that is if you are actually documenting the specifics of how those control loops are implemented.